### ISSUE TITLE
Changes in tags do not show up in history anymore

### ISSUE THREAD

**Issue by ilpssun** [ixti/redmine_tags#173]
Maybe I remember incorrectly but as far as I recall changes to tags were documented in the history of an issue.  Since upgrading the plugin to 3.2.0 (for bulk editing) this does not seem to be the case anymore.  It would be very helpful, however, to see who changed the tags on an issue.  Please restore this functionality.

**Comment by ilpssun** [comment]
The issue persists in Redmine 3.4.2 (plugin version 3.2.0).  Can anybody confirm this issue or do we have some kind of misconfiguration in our setup?

**Comment by PowerKiKi** [comment]
I can confirm that tag changes do not appear in the issue history anymore in the following environnement. I am pretty it used to work a long time ago.

```
Environment:
  Redmine version                3.4.3.stable
  Ruby version                   2.3.1-p112 (2016-04-26) [x86_64-linux-gnu]
  Rails version                  4.2.8
  Environment                    development
  Database adapter               Mysql2
SCM:
  Subversion                     1.9.3
  Bazaar                         2.7.0
  Git                            2.7.4
  Filesystem                     
Redmine plugins:
  redmine_bootstrap_kit          0.2.4
  redmine_github_hook            2.2.0
  redmine_image_clipboard_paste  3.3.0
  redmine_tags                   3.2.0
```

**Comment by marius-balteanu** [comment]
Indeed, I've introduced this bug when I made the bulk edit feature. I'll take a look these days.

**Comment by marius-balteanu** [comment]
Fixed in version 3.2.1 with a test.
